superset-frontend/src/SqlLab/actions/sqlLab.js

```diff
@@ -1273,6 +1273,11 @@ export function popSavedQuery(saveQueryId) {
       endpoint: `/savedqueryviewapi/api/get/${saveQueryId}`,
     })
       .then(({ json }) => {
+        // if description from result is 'None', need to omit this property.
+        if(json.result.description === 'None') {
```

Comment:
mm, what if I set the description as None?
Unlikely, but in any case, possible.

I think the problem needs to be fixed on the api side, None is being converted to string when in fact it should be translated to null or undefined.